# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1390668482 18000
#      Sat Jan 25 11:48:02 2014 -0500
# Branch stable
# Node ID af04b9d9f7ed5c603e27d10e2ef8fc1cfa07dc84
# Parent  4da51757372f51c1f725b271493057ab784dba3b
allow urlwrite to work again

* urlwrite.cc (Furlwrite): Reverse sense of test when checking whether
url_transfer object is valid.

diff --git a/libinterp/corefcn/urlwrite.cc b/libinterp/corefcn/urlwrite.cc
--- a/libinterp/corefcn/urlwrite.cc
+++ b/libinterp/corefcn/urlwrite.cc
@@ -413,17 +413,17 @@ urlwrite (\"http://www.google.com/search
     }
 
   unwind_protect_safe frame;
 
   frame.add_fcn (delete_file, filename);
 
   url_transfer curl = url_transfer (url, ofile);
 
-  if (! curl.is_valid ())
+  if (curl.is_valid ())
     {
       curl.http_action (param, method);
 
       ofile.close ();
 
       if (curl.good ())
         frame.discard ();
 
